src/plugins/list-plugin.tsx

```diff
@@ -46,5 +50,23 @@ export function registerListBlocks() {
 
 export function withListBlocks(editor: Editor) {
   registerListBlocks();
+
+  registerToolbarButtons(editor, [
+    {
+      format: EFormat.numberedList,
+      SvgIcon: IconNumberedList,
+      tooltip: getPlatformTooltip("numbered list"),
+      isActive: () => !!editor && isBlockActive(editor, EFormat.numberedList),
+      onClick: () => toggleBlock(editor, EFormat.numberedList)
```

Comment:
I notice you're checking `!!editor` in similar places in `core-blocks-plugin.tsx` and `core-marks-plugin.tsx`. Can those be removed, or are similar checks necessary here?

Reply:
Good catch -- given that an `editor` is required to register the handler, I think we can remove the checks everywhere.